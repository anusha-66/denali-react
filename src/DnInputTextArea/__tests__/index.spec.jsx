/**
 * Copyright 2020, Verizon Media
 * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
 */

import React from 'react';
import { render } from 'enzyme';
import { DnInputTextArea } from '../../index';

describe('DnInputTextArea Tests', () => {
    it('should render with default props', () => {
        expect.assertions(1);

        const rendered = render(<DnInputTextArea />);

        expect(rendered).toMatchSnapshot();
    });

    it('should render correctly with custom props', () => {
        expect.assertions(1);

        const rendered = render(
            <div className="Foo">
                <DnInputTextArea
                    id="someId"
                    isActive
                    isDisabled
                    isError
                    isInverse
                    errorMessage="Error"
                    placeholderText="Just an input field"
                />
            </div>,
        );

        expect(rendered).toMatchSnapshot();
    });
});
