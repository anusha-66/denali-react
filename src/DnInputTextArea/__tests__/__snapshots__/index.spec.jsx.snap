// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DnInputTextArea Tests should render correctly with custom props 1`] = `
<div
  class="Foo"
>
  <div
    class="input is-active is-inverse is-disabled is-error"
  >
    <textarea
      disabled=""
      id="someId"
      placeholder="Just an input field"
    />
    <p
      class="message"
    >
      Error
    </p>
  </div>
</div>
`;

exports[`DnInputTextArea Tests should render with default props 1`] = `
<div
  class="input"
>
  <textarea />
</div>
`;
